server/fly-away/src/main/java/com/codestates/flyaway/web/board/BoardController.java

```diff
@@ -0,0 +1,70 @@
+package com.codestates.flyaway.web.board;
+
+import com.codestates.flyaway.domain.board.entity.Board;
+import com.codestates.flyaway.domain.board.service.BoardService;
+import com.codestates.flyaway.global.dto.MultiResponseDto;
+import com.codestates.flyaway.global.dto.SingleResponseDto;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Page;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+import static com.codestates.flyaway.web.board.dto.BoardDto.*;
+
+@RestController
+@Slf4j
+@RequiredArgsConstructor
+@RequestMapping("/board")
+public class BoardController {
+
+    private final BoardService boardService;
+
+    @PostMapping
+    public ResponseEntity createArticle(@Validated @RequestBody Create createDto){
+
+        BoardResponseDto created = boardService.create(createDto);
+
+        return new ResponseEntity(new SingleResponseDto(created), HttpStatus.CREATED);
+    }
+
+    @PatchMapping("/{boardId}/update")
```

Comment:
```suggestion
    @PatchMapping("/{boardId}")
```
Patch 메서드로 명시했다면 update 라는 표현은 의미가 없는 것 같습니다.